docs/content/preview/deploy/manual-deployment/install-software.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Install YugabyteDB software
-headerTitle: Install software
+title: Manual software installation
+headerTitle: Manual software installation
```

Comment:
Like description and headcontent, title and headerTitle are related but don't have to be the same. headerTitle is the title displayed on the page, title is for SEO; headerTitle should match or be similar to linkTitle. 
so headerTitle of "Install software" or "Install YugabyteDB software" would work here